plugins/woocommerce-admin/client/marketplace/components/product-list-content/no-results.scss

```diff
@@ -19,6 +20,7 @@
 .woocommerce-marketplace__no-results__description {
 	text-align: center;
 	font-size: 13px;
+	max-width: 52ch;
```

Comment:
TIL about `ch`. Apparently it can be [unpredictable](https://meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit/) with non monospaced fonts – would `em` be a safer alternative?

Reply:
No worries, clearly this is a conscious choice.